ui/package.json

```diff
@@ -12,7 +12,9 @@
     "@pingcap-incubator/keyvis": "^0.0.8",
     "@pingcap-incubator/statement": "^0.0.8",
     "@umijs/hooks": "^1.8.0",
-    "antd": "^3.26.5",
+    "@ant-design/compatible": "^1.0.0",
```

Comment:
prefer not to introduce the compatible layer.

Reply:
@breeswish  @baurine  I fixed all compatible code except the one `LegacyIcon` in `src/layout/main/Sider/index.js`. antd4 import icon as need, should I use a `switch` to check all app.icon ? Any good ideas?

Reply:
hey @SSebo, I think you can try use component instead of string as the value of app.icon, likes:

```
export default {
  id: 'statement',
  routerPrefix: '/statement',
  icon: <ThunderboltOutlined />, // or () => <ThunderboltOutlined/>
  translations,
}
```

Reply:
@baurine where to import `<ThunderboltOutlined/>` ?

Reply:
Does it not exist in '@ant-design/icons' (I don't try...)

Reply:
It is the same as other icons:

```js
import { ThunderboltOutlined } from '@ant-design/icons';
```

Reply:
Sorry, I mean do not allow to return React object in modules export.

Reply:
```js
import { ThunderboltOutlined } from '@ant-design/icons';

export default {
  id: 'statement',
  routerPrefix: '/statement',
  icon: ThunderboltOutlined,
  translations,
}
```